mllib/src/main/scala/org/apache/spark/ml/r/SparkRWrappers.scala

```diff
@@ -51,6 +52,22 @@ private[r] object SparkRWrappers {
     pipeline.fit(df)
   }
 
+  def fitKMeans(
+      df: DataFrame,
+      initMode: String,
+      maxIter: Double,
+      k: Double,
+      columns: Array[String]): PipelineModel = {
+    val assembler = new VectorAssembler().setInputCols(columns).setOutputCol("temp-features")
```

Comment:
Do not name the output column. There exists a chance that the output column name already exists. The default one should work well. In L66, use `setFeaturesCol(assembler.getOutputCol)`.